Control/Monad/Primitive.hs

```diff
@@ -69,26 +68,26 @@ import qualified Control.Monad.Trans.RWS.Strict    as Strict ( RWST   )
 import qualified Control.Monad.Trans.State.Strict  as Strict ( StateT )
 import qualified Control.Monad.Trans.Writer.Strict as Strict ( WriterT )
 
--- | Class of monads which can perform primitive state-transformer actions
+-- | Class of monads which can perform primitive state-transformer actions.
 class Monad m => PrimMonad m where
-  -- | State token type
+  -- | State token type.
   type PrimState m
 
-  -- | Execute a primitive operation
+  -- | Execute a primitive operation.
   primitive :: (State# (PrimState m) -> (# State# (PrimState m), a #)) -> m a
 
 -- | Class of primitive monads for state-transformer actions.
 --
--- Unlike 'PrimMonad', this typeclass requires that the @Monad@ be fully
+-- Unlike 'PrimMonad', this typeclass requires the @Monad@ to be fully
```

Comment:
Is this really an improvement?

Reply:
I wasn't sure if the original version is proper english, but apparently it is. I'll change this back.